http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfigurationBuilder.java

```diff
@@ -199,7 +199,10 @@ protected OidcClientConfiguration internalBuild(final OidcJsonConfiguration oidc
 
         oidcClientConfiguration.setTokenSignatureAlgorithm(oidcJsonConfiguration.getTokenSignatureAlgorithm());
 
-        String tmpLogoutPath = System.getProperty(LOGOUT_PATH);
+        String tmpLogoutPath = oidcJsonConfiguration.getLogoutPath();
+        if (tmpLogoutPath == null) {
+            tmpLogoutPath = System.getProperty(LOGOUT_PATH);
```

Comment:
The system property configuration shouldn't be part of this branch. In this 2.x branch, logout should be configured using OIDC JSON configuration only.

Reply:
done